source/fuzz/fuzzer_pass_donate_modules.cpp

```diff
@@ -413,14 +413,24 @@ void FuzzerPassDonateModules::HandleTypeOrValue(
             argument_type_ids));
       }
     } break;
+    case SpvOpSpecConstantOp: {
+      // Make sure the module contains an OpConstant instruction
+      // that has suitable type.
+      FindOrCreateZeroConstant(type_or_value.type_id());
```

Comment:
I believe you should assign the result of this call to `new_result_id`, so that the OpSpecConstantOp gets remapped to the id of the zero constant.

Reply:
Fixed in e31131c.